### DISCUSSION TITLE
Normalization before or after cell subset?

### DISCUSSION THREAD

**Discussion by moramassaro** [satijalab/seurat#4656] [Q&A]
Hi everyone. I would like to cluster a subset of cells of my sc data. Therefore I would like to know if you recommend doing the normalization before or after my cell subset. Thank you in advance

**Comment by saketkc** [comment]
For log-normalization it would result in more or less similar results either way. If you are using SCTransform, you should normalize each experiment individually.

**Reply by moramassaro** [reply]
Thank you! So, the thing is I have a dataset from ~300k cells already clustered in three mayor groups (immune cells, epithelial cells and el otro). I've already subset epithelial cells and I'm interested on 4 subgroups of them. I want to run a pseudotime on those cells since they are lineage related. Should I subset those 4 groups first and then run SCT or run SCT on the whole epithelial dataset and then subset the groups?
